graphql/data/schema.js

```diff
@@ -170,6 +170,7 @@ const backgroundImageType = new GraphQLObjectType({
       type: GraphQLString,
       description: 'The image file URL',
     },
+    category: { type: GraphQLString, description: 'the image file URL' },
```

Comment:
We should make this NonNull. This is the approach I'm thinking:
* The existing background images in the DB might have an undefined category (we don't migrate a default value).
* However, the BackgroundImage *model* sets a default category to something like "original" or "default" -- so we can know the category field will always exist when calling the API.
* For backwards compatibility, we can't rely on the "category" being passed by the client in the mutation. So we fall back to that "default" category when filtering on images.
* (In this structure, the "all" naming you have here is a little misleading, in that it suggests we ignore category altogether.)